### PR TITLE
Integration tests port fix, logs output, lib share changes

### PR DESCRIPTION
**Pull request by egor-ryashin**
This PR makes it easier to debug integration tests failures during *Travis* build. Also, made a JVM parameters fix for `supervisord` (the parameters were ignored before that).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,3 +42,11 @@ matrix:
         - mvn install -q -ff -DskipTests -B
       script:
         - $TRAVIS_BUILD_DIR/ci/travis_script_integration.sh
+      after_failure:
+        - for v in /shared/logs/*.log ; do
+          echo $v logtail ======================== ; tail -100 $v ;
+          done
+        - for v in broker middlemanager overlord router coordinator historical ; do
+          echo $v dmesg ======================== ;
+          docker exec -it druid-$v sh -c 'dmesg | tail -3' ;
```
Comment (himanshug) [inline .travis.yml]: is it possible to indent the lines inside for loop?

**Comment by himanshug** [comment]
:+1: aside from minor comment
@egor-ryashin have you tested this by introducing an artificial test error in integration test build to make it fail and see the logs produced in build logs? it would be nice to check that things work as expected.

File: .travis.yml (hunk 1 above)
Comment (egor-ryashin) [reply .travis.yml]: Nope.

File: integration-tests/docker/broker.conf [hunk 2]
```diff
@@ -21,7 +21,7 @@ command=java
   -Ddruid.broker.cache.populateCache=true
   -Ddruid.cache.type=local
   -Ddruid.cache.sizeInBytes=40000000
-  -cp /usr/local/druid/lib/*
+  -cp /shared/docker/lib/*
```
Comment (b-slim) [inline integration-tests/docker/broker.conf]: not sure why we have this change? thought libs are always `/usr/local/druid/lib/*`?

Comment (egor-ryashin) [reply integration-tests/docker/broker.conf]: It makes it easier to recompile libs without rebuilding the image, when you fixing/debugging tests locally. For example, if you re-run tests multiple times locally then rebuilding the image is not required and time-consuming.

**Comment by egor-ryashin** [comment]
@himanshug Yes, tested with error, the output is ok.

**Comment by himanshug** [comment]
@egor-ryashin thanks for checking.

**Pull request merged**